### ISSUE TITLE
Review all check in google fonts profile

### ISSUE THREAD

**Issue by m4rc1e** [googlefonts/fontbakery#4102]
We should do an annual review to see if any checks are no longer relevant. I reckon many of them are now redundant since some of them exist only to keep Win XP users happy.

Keeping redundant checks significantly slow us down since foundries/designers often question them. This entails more admin from us.

I've just skimmed the fontbakery profile and I reckon the following are redundant (I could be wrong)

- com.google.fonts/check/kerning_for_non_ligated_sequences: Idk of any apps which do not have a liga feature?
- com.google.fonts/check/epar: Never seen a font with this table and probably never will
- 'com.google.fonts/check/unitsperem_strict': You can have any arbitrary upm you want. We shouldn't restrict people

**Comment by felipesanches** [comment]
I agree EPAR check should be deprecated, unless @dberlow and/or @davelab6 vouch for keeping it. In that case though, I think there should be some complementary action towards promoting the table, otherwise it seems it will continue to be ignored forever. If only users had a better sense of what benefit does one get from having that table...

**Comment by yanone** [comment]
I propose to delete `com.google.fonts/check/description/eof_linebreak` and possibly other similar checks. The formatting of HTML is exactly not defined by whitespace, so it shouldn't be enforced or even warned.

I also propose to alter `com.google.fonts/check/license/OFL_body_text` so that it doesn't check for wrong line ending any longer, when the rest of the body text is otherwise correct. It's so easy to accidentally get that wrong. A line ending does nothing to the legal status of the license except waste time of everyone involved.

I also propose to alter `com.google.fonts/check/math_signs_width` to only warn in case the font contains table figures, either by default or by feature activation. When figures are proportional, math symbols have zero need to be of identical width. In case of the default figures being proportional and table figures being activatable, I propose to check for math sign width also only under the application of the `tnum` feature. I would use Harfbuzz for that for convenience instead of trying to read that information directly from the font.

I know that the softhyphen check is being discussed elsewhere. But the .notdef being empty should also be a FAIL for GF. I thought it was, but that font I worked on today didn't fail for that.

**Comment by khaledhosny** [comment]
> com.google.fonts/check/kerning_for_non_ligated_sequences: Idk of any apps which do not have a liga feature?

This is useful when the user disabled `liga` (either globally or for specific sequence, e.g. German combined words which many users consider ligatures across them to be wrong).

**Comment by RosaWagner** [comment]
Some of the proposals here are having open issues:

- [ ] https://github.com/googlefonts/fontbakery/issues/4095
- [ ] https://github.com/googlefonts/fontbakery/issues/4094
- [ ] https://github.com/googlefonts/fontbakery/issues/4093
- [ ] https://github.com/googlefonts/fontbakery/issues/4092
